platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/address/AddressBook.java

```diff
@@ -543,6 +528,49 @@ public Iterator<Address> iterator() {
         return new AddressBookIterator(orderedNodeIds.iterator(), addresses);
     }
 
+    /**
+     * The text form of an address book that appears in config.txt
+     *
+     * @return the string form of the AddressBook that would appear in config.txt
+     */
+    public String toConfigText() {
+        final StringBuilder sb = new StringBuilder();
+
+        for (final Address address : this) {
+            sb.append("  ").append(address.toConfigText()).append(",\n");
+        }
+
+        return sb.toString();
+    }
```

Comment:
There is a utility called `TextTable` that would make the formatting of the address book a lot cleaner by lining up the columns. It would look something like this:

```suggestion
    public String toConfigText() {
        final TextTable table = new TextTable().setBordersEnabled(false);

        for (final Address address : this) {
            table.addRow(
                "address,"
                address.getNickname() + ",",
                address.getSelfName() + ",",
                address.getStake() + ",",
                ipString(address.getAddressInternalIpv4()) + ",",
                address.getPortInternalIpv4() + ",",
                pString(address.getAddressExternalIpv4()) + ",",
                address.getPortExternalIpv4());
        }

        return table.render();
    }
```

Reply:
I have adopted the use of TextTable in a new commit.